packages/adapter-node/README.md

```diff
@@ -16,6 +16,7 @@ export default {
 			// default options are shown
 			out: 'build',
 			precompress: false,
+			// Name of the environment variables from which host and port are read
```

Comment:
I think this was already explained at https://github.com/sveltejs/kit/tree/master/packages/adapter-node#env. I agree that it's not too clear from the existing wording though, so we should add the clarification there instead.